src/coreclr/jit/block.cpp

```diff
@@ -231,9 +236,13 @@ FlowEdge* Compiler::BlockPredsWithEH(BasicBlock* blk)
                 {
                     res = new (this, CMK_FlowEdge) FlowEdge(filterBlk, blk, res);
 
-                    assert(filterBlk->VisitEHEnclosedHandlerSecondPassSuccs(this, [blk](BasicBlock* succ) {
-                        return succ == blk ? BasicBlockVisit::Abort : BasicBlockVisit::Continue;
-                    }) == BasicBlockVisit::Abort);
+                    assert(filterBlk->VisitEHEnclosedHandlerSecondPassSuccs(this,
+                                                                            [blk](BasicBlock* succ)
+                                                                            {
+                                                                                return succ == blk
+                                                                                           ? BasicBlockVisit::Abort
+                                                                                           : BasicBlockVisit::Continue;
+                                                                            }) == BasicBlockVisit::Abort);
```

Comment:
This lambda formatting change looks like a significant regression... Maybe turn https://clang.llvm.org/docs/ClangFormatStyleOptions.html#bracewrapping, `BeforeLambdaBody`, to `false`? Or https://clang.llvm.org/docs/ClangFormatStyleOptions.html#lambdabodyindentation to `OuterScope`?

Reply:
I set both of these as you suggest, and it looks pretty good to me.